### PR TITLE
Fix vscode-neovim.mouseSelectionStartVisualMode

### PR DESCRIPTION
**Pull request by cspotcode**
It looks like #334 accidentally ignored `vscode-neovim.mouseSelectionStartVisualMode`.

Explained here: https://github.com/asvetliakov/vscode-neovim/issues/395#issuecomment-897005222

This PR re-enables that setting.

### CODE REVIEW

**Comment by yogeshdhamija** [comment]
Confirmed working on my machine 👍🏽  ❤️

**Review by yogeshdhamija — APPROVED** [review]

**Comment by trkoch** [comment]
I was also curious about that `mouseSelectionStartVisualMode` setting. Good catch!

**Comment by sky0hunter** [comment]
It is working partially for me; The selection does translate to the neovim selection, but on some files the selection always starts from the end of the file, or instead of starting a new selection it uses the current cursor position as starting point.

**Comment by searleser97** [comment]
> It is working partially for me; The selection does translate to the neovim selection, but on some files the selection always starts from the end of the file, or instead of starting a new selection it uses the current cursor position as starting point.

I have the same problem

**Comment by trkoch** [comment]
I noticed double click selects a word and the following whitespace (including the new line character).

File: .vscode/tasks.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
                         "regexp": "Compiling.*?|Compilation .*?starting"
                     },
                     "endsPattern": {
-                        "regexp": "Compiled .*?successfully|Compilation .*?finished"
+                        "regexp": "webpack \\d+\\.\\d+\\.\\d+ compiled|Compilation .*?finished"
```
Comment (trkoch) [inline .vscode/tasks.json:44]: Why was this change necessary? Can you elaborate on this?

Comment (auderer) [reply .vscode/tasks.json:44]: this seems unrelated to me, @cspotcode can you remove this from the PR?

Comment (cspotcode) [reply .vscode/tasks.json:44]: I don't remember entirely, but I think it was in response to some changes to webpack's logging.  Rather than logging "Compiled * successfully" it was logging "webpack <version #> compiled."  The regexp required an update to match.

**Comment by JimmyMcBride** [comment]
> It is working partially for me; The selection does translate to the neovim selection, but on some files the selection always starts from the end of the file, or instead of starting a new selection it uses the current cursor position as starting point.

Same issue. It seems the more you highlight, the more it selects in the opposite direction. It also includes the block on the end. I could live with it including the block on the end (even though it doesn't show it highlighted) but having it select more characters from the opposite direction is kind of breaking.

**Comment by auderer** [comment]
> > It is working partially for me; The selection does translate to the neovim selection, but on some files the selection always starts from the end of the file, or instead of starting a new selection it uses the current cursor position as starting point.
> 
> Same issue. It seems the more you highlight, the more it selects in the opposite direction. It also includes the block on the end. I could live with it including the block on the end (even though it doesn't show it highlighted) but having it select more characters from the opposite direction is kind of breaking.

We should make a new issue for this. This PR just fixes the problem where the user's settings for mouseSelection are ignored, let's get this merged then discuss improving the mouse behavior.

**Review by theol0403 — APPROVED** [review]

**Comment by theol0403** [comment]
Thanks for the investigation and PR! Yeah, please create another issue for this.
